### ISSUE TITLE
I can't find it  ==> fine_tune.py

### ISSUE THREAD

**Issue by knightzz1998** [DeepGraphLearning/SPN#5]
3. Train a CRF-GAT model

python fine_tune.py --dataset pubmed \
    --no-proxy \
    --refine-epochs 5000 \
    --refine-eval-every 50 \
    --no-log-softmax \
    GAT
Hello, is there something wrong here, I can't find it in the repository the file fine_tune.py

**Comment by mnqu** [comment]
Thanks for your interest and for pointing out the problem.
The fine_tune.py should actually be refine.py, and we have fixed the README file.
